renorm: intermittent fp16 accuracy failure in daily CI

Two `test_renorm` cases fail intermittently on fp16 in the daily CI (quick-cpu mode):

```
FAILED tests/test_renorm.py::test_renorm[1-3-dtype0-shape1] - AssertionError: Tensor-likes are not close!
FAILED tests/test_renorm.py::test_renorm[-1-2-dtype0-shape0] - AssertionError: Tensor-likes are not close!
```

Both failures are `dtype0` = `torch.float16` only. Other dtypes pass. Likely a marginal numerical precision issue triggered by specific random inputs.

Reference: https://github.com/flagos-ai/FlagGems/actions/runs/27878424327/job/82501732630

--- a/src/flag_gems/ops/renorm.py
+++ b/src/flag_gems/ops/renorm.py
@@ -113,7 +113,7 @@ def renorm(input, p, dim, maxnorm):
         N = input.numel() // M
 
         input = input.contiguous()
-        norms = torch.empty((M,), dtype=input.dtype, device=input.device)
+        norms = torch.empty((M,), dtype=torch.float32, device=input.device)
 
         BLOCK = min(triton.next_power_of_2(N), 128)
         grid = (M,)
@@ -167,7 +167,7 @@ def renorm_(input, p, dim, maxnorm):
         N = input.numel() // M
 
         input = input.contiguous()
-        norms = torch.empty((M,), dtype=input.dtype, device=input.device)
+        norms = torch.empty((M,), dtype=torch.float32, device=input.device)
 
         BLOCK = min(triton.next_power_of_2(N), 128)
         grid = (M,)

